sdk/keyvault/keyvault-admin/test/README.md

```diff
@@ -0,0 +1,42 @@
+# Testing
+
+To test this project, make sure to build it by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building), then follow the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/test-resources.json) that already has all of the the necessary configurations.
+
+The Azure resource that is used by the tests in this project is:
+
+- An [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/general/basic-concepts). Your Azure application needs to be assigned as the **owner** of this Azure Key Vault. The steps are provided [below](#AAD-based-authentication).
+
+To run the live tests, you will also need to set the below environment variables:
+
+- `TEST_MODE`: Should have `live` assigned.
+- `AZURE_CLIENT_ID`: The client ID of an Azure Active Directory application.
+- `AZURE_CLIENT_SECRET`: The client secret of an Azure Active Directory application.
+- `AZURE_TENANT_ID`: The Tenant ID of your organization in Azure Active Directory.
+- `KEYVAULT_URI`: The name of the key vault to use in the samples.
+- `KEYVAULT_NAME`: The name of the KeyVault to use.
+- `BLOB_STORAGE_URI`: URI of the Blob Storage instance, with the name of the container where the Key Vault backups will be generated.
+- `BLOB_STORAGE_SAS_TOKEN`: URI of the Blob Storage instance, with the name of the container where the Key Vault backups will be generated.
+- `CLIENT_OBJECT_ID`: Object ID of the application, tenant or principal to whom the role will be assigned to.
```

Comment:
TODO: Mention that this shouldn't the Object ID of the principal used to authenticate. Or you'll kick yourself out of the system. At least in the tests.